priv/templates/components/button.eex

```diff
@@ -234,11 +240,21 @@ defmodule <%= @module %> do
       }
       {drop_rest(@rest)}
     >
+      <!-- loading at the start -->
+      <%%= for loader <- @loading, loader[:position] == "start" do %>
+        <%%= render_slot(loader) %>
+      <%% end %>
```

Comment:
Please use `:for=` for all for you used in this file, if you have no html tag so I think it is better to create `div` or `span`